frontend/src/routes/(app)/project/[owner]/[project]/chart/[chartIndex=integer]/+page.svelte

```diff
@@ -12,7 +13,7 @@
 
 	export let data;
 
-	let isChartEdit = false;
+	let isChartEdit = $page.url.searchParams.get('edit') ? true : false;
```

Comment:
If we do this via the URL, we should probably update the URL in the EditHeader as well instead of updating this variable probably. Then it works on reload as well.